includes/express-checkout/class-wc-payments-express-checkout-ajax-handler.php

```diff
@@ -48,26 +48,38 @@ public function init() {
 
 	/**
 	 * Create order. Security is handled by WC.
+	 *
+	 * @throws Exception If cart is empty.
```

Comment:
That is needed because we have a PHPCS rule enforcing that. I updated the comment in [45e3145](https://github.com/Automattic/woocommerce-payments/pull/9108/commits/45e3145d69e69c4950a3a887bd78c877e86500da) to mention that is handled within the method 👍